lib/mix/lib/mix/tasks/compile.ex

```diff
@@ -72,7 +71,7 @@ defmodule Mix.Tasks.Compile do
       default case with dependencies)
     * `--no-prune-code-paths` - do not prune code paths before compilation, this keeps
       the entirety of Erlang/OTP available when the project starts
-    * `--no-protocol-consolidation` - skips protocol consolidation
```

Comment:
That's a breaking change. Shouldn't both `no-protocol-consolidation` and `no-consolidate-protocols` be supported?